assimilation_code/modules/utilities/mpi_utilities_mod.f90

```diff
@@ -50,7 +50,7 @@ module mpi_utilities_mod
 ! For more help on compiling a module which uses MPI see the 
 ! $DART/developer_tests/mpi_utilities/tests/README
 
-use mpi
+use mpi_f08
```

Comment:
maybe add a comment before this line for the user, saying if you don't have the 2008 version of mpi available, try 'use mpi' instead?